src/lib/message/L2ToL1Message.ts

```diff
@@ -188,45 +192,50 @@ export class L2ToL1Message {
 }
 
 /**
- * Provides read-only access for l2-to-l1-messages
+ * Provides read-only access for Chain-to-ParentChain-messages
  */
-export class L2ToL1MessageReader extends L2ToL1Message {
+export class ChildToParentMessageReader extends ChildToParentMessage {
   private readonly classicReader?: classic.L2ToL1MessageReaderClassic
   private readonly nitroReader?: nitro.L2ToL1MessageReaderNitro
 
   constructor(
-    protected readonly l1Provider: Provider,
-    event: L2ToL1TransactionEvent
+    protected readonly parentChainProvider: Provider,
+    event: ChildToParentTransactionEvent
   ) {
     super()
     if (this.isClassic(event)) {
       this.classicReader = new classic.L2ToL1MessageReaderClassic(
-        l1Provider,
+        parentChainProvider,
         event.batchNumber,
         event.indexInBatch
       )
     } else {
-      this.nitroReader = new nitro.L2ToL1MessageReaderNitro(l1Provider, event)
+      this.nitroReader = new nitro.L2ToL1MessageReaderNitro(
+        parentChainProvider,
+        event
+      )
     }
   }
 
   public async getOutboxProof(
-    l2Provider: Provider
+    chainProvider: Provider
```

Comment:
i see that some are `childChainProvider` on this file and some are `chainProvider`
i prefer the former for all names